Documentation/Basics/GeneralConventions/GuidelinesForImages.rst

```diff
@@ -9,167 +9,58 @@
-common are:
+For accessibility reasons always provide an alt text.
 
-#. Screenshots
-#. Screenshots, augmented with graphic elements, the most common being
-   arrows, texts, boxes and numbers to highlight parts of the screenshot,
-   add extra explanation etc., with the intention of making the image more
-   useful for the reader
-#. Diagrams, such as UML diagrams
-
-
-.. index:: Images; Style guide
-
-General guidelines
-==================
-
-*  In general, comply with the
-   `TYPO3 web style guide <https://styleguide.typo3.org/patternlab/public/index.html>`__
-
-
-.. index:: Images; Fonts
-
-Texts & fonts
--------------
-
-*  When using fonts, please use recommended: **Primary font: "Source Sans Pro", "Helvetica", "Arial",
-   sans-serif;**. You can download "Source Sance Pro" as TTF files:
-   `Assets <https://styleguide.typo3.org/Data/TYPO3_Assets_2017-08-23.zip>`__,
-   path: :file:`00\ -\ Assets/Fonts/SourceCodePro-Regular.ttf`
-*  If you use text in an SVG, make sure it can be
-   displayed on systems that have not installed the used font.
-   There are a
-   number of ways to do this. If in doubt, ask for :ref:`help <help>` or save the image
-   as bitmap (.png). However, if SVG is used, the image can be scaled
-   better.
-*  In any case, you should be safe with using a commonly supported font
-   like Helvetica or Arial. Remember, the Web Style Guide proposes
-   *Source Sans Pro, Helvetica, Arial*.
-
-
-.. index::
-   Images; Formats
-   Images; PNG
-   Images; SVG
+.. _guidelines-for-images-formats:
 
 Image formats
 -------------
 
 *  It is recommended to use PNG for bitmaps (for example screenshots, photographs)
    and SVG for vector graphics images. In any case, you can use .png.
 
-.. index:: Screenshots; Guidelines
+
+.. _guidelines-for-images-screenshot:
 
 Guidelines for screenshots
 ==========================
 
-*  use PNG format (.png file ending)
-*  Do not use huge full screen screenshots (except in some cases where it makes sense,
-   for example when describing the various parts of the backend). Select a portion
-   of a screen that will reflect what you are trying to show. To save yourself hassle,
-   use a screenshot tool which can create a screenshot of a portion of a screen.
-
-Here are some :ref:`examples for screenshots <t3contribute:how-to-see-merge-conflicts>`.
-
-
-
-.. code-block:: rst
-
-   .. image:: images/github-getting-started-raw.png
-      :class: with-shadow
-      :scale: 70
-
-How it looks:
-
-.. image:: images/github-getting-started-raw.png
-   :class: with-shadow
-   :scale: 70
-
-Another example:
-
-.. image:: images/backend-raw.png
-   :class: with-shadow
-   :scale: 70
-
-
-You may however notice, that it might be helpful to point out some relevant part of the
-screenshots. That is what we do by creating images as described in the next section.
-
-
-.. index:: Screenshots; Adding graphics elements
+..  note::
+    We do not have an automatic screenshot tool working with current TYPO3
```

Comment:
do we really need to mention this? i would prefer we would only sate the IS state and not mention the was state.

Reply:
as discussed,. we keep this in as this is a common question that needs to be answered